Duckling/Distance/IT/Corpus.hs

```diff
@@ -0,0 +1,47 @@
+-- Copyright (c) 2016-present, Facebook, Inc.
+-- All rights reserved.
+--
+-- This source code is licensed under the BSD-style license found in the
+-- LICENSE file in the root directory of this source tree. An additional grant
+-- of patent rights can be found in the PATENTS file in the same directory.
+
+
+{-# LANGUAGE OverloadedStrings #-}
+
+module Duckling.Distance.IT.Corpus
+  ( corpus ) where
+
+import Prelude
+import Data.String
+
+import Duckling.Distance.Types
+import Duckling.Locale
+import Duckling.Resolve
+import Duckling.Testing.Types
+
+corpus :: Corpus
+corpus = (testContext {locale = makeLocale IT Nothing}, testOptions, allExamples)
+
+allExamples :: [Example]
+allExamples = concat
+  [ examples (simple Kilometre 3)
+             [ "3 kilometri"
```

Comment:
"kilometri" is not an italian word 😅

Reply:
Actually it is 😄
"Chilometri" is more used than "Kilometri", which is still a valid italian word:

Vocabulary:
http://www.treccani.it/vocabolario/km/

Explanation:
http://www.treccani.it/magazine/lingua_italiana/domande_e_risposte/grammatica/grammatica_253.html

Reply:
>  viceversa, in forma piena «è preferibile ricorrere alla grafia italianizzata» (chilogrammo, chilometro, chilolitro, chilowatt).

So outrageous.